module-gui/gui/widgets/BoxLayout.hpp

```diff
@@ -141,12 +141,12 @@ namespace gui
     class VBox : public BoxLayout
     {
       public:
-        void resizeItems() override;
+        virtual void resizeItems() override;
```

Comment:
this should trigger warn on clang-tidy, doesn't it?

Reply:
it does, fixed